distributed/comm/ucx.py

```diff
@@ -116,12 +132,7 @@ def peer_address(self) -> str:
             # Send frames
             for frame in frames:
                 if nbytes(frame) > 0:
-                    if hasattr(frame, "__array_interface__") or hasattr(
-                        frame, "__cuda_array_interface__"
-                    ):
-                        await self.ep.send(frame)
-                    else:
-                        await self.ep.send(frame)
+                    await self.ep.send(frame)
```

Comment:
We can do this now becuase ucx-py is doing the inspection on the type of frame, correct ?

Reply:
Thinking about this more, what happens when the frame is 0 length ?  current ucx-py doesn't support zero length:

https://github.com/rapidsai/ucx-py/blob/e23c4137e370a2394a837b36f6a75274c23f579f/ucp/_libs/utils.pyx#L47

Reply:
ucx-py reads the `frame` usng `__array_interface__`, `__cuda_array_interface__`, and the buffer protocol. 

And yes, we need the `if nbytes(frame) > 0:` check because ucx-py doesn't support 0 length frames

Reply:
Right!  That make sense